src/typeahead/typeahead.js

```diff
@@ -419,6 +421,8 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position', 'ui.bootstrap
 
         if (appendToBody) {
           $document.find('body').append($popup);
+        } else if(appendToElementId !== false) {
+            $document.find('#' + appendToElementId).append($popup);
```

Comment:
It should be noted that this library does not have a dependency on jQuery, so this would fail with purely jqLite.